podvm/addons/README.md

```diff
@@ -0,0 +1,21 @@
+## Introduction
+
+The addons directory is used to enable different addons for the podvm image.
+Each addon and its associated files (binaries, configuration etc) should be under
+specific sub-dir under `addons`. 
+
+Each addon sub-dir needs to have `setup.sh` for setting up the addon.
+To enable the addon create a file named `.enable` in the top level podvm/addons
+directory and in the specific addon sub-dir.
+
+The addons are setup by the `qcow2/setup_addons.sh` script.
+
+An example skeleton addon is provided as an example
+
+By default all addons are disabled. If you want to enable an addon, then you will need to do the following:
+
+```
+export ADDON_NAME=REPLACE_ME
+touch podvm/addons/.enable
+touch podvm/addons/${ADDON_NAME}.enable
```

Comment:
Do we need to introduce this `.enable` mechanism or can we cover this with makefile flags?

Reply:
Possible to cover with makefile. Like for policy, based on AGENT_POLICY=yes, these files will be created using instructions in the Makefile.

Reply:
https://github.com/confidential-containers/cloud-api-adaptor/pull/1607/files#diff-6bc6c17c0f0e3f8d3827c3664d191076319cc9059c50dee659ea9392344ed602R38-R41

Reply:
Similarly, for addons which we decide to enable always, we can add an option in makefile to create the `.enable` file and include it.

I'm thinking the same approach can be used when switching to mkosi although I'm not yet sure how.